rust/Cargo.toml

```diff
@@ -178,6 +178,7 @@ url = "2.3"
 walkdir = "2"
 warp = "0.3"
 which = "4.3"
+ya-gcp = { version = "0.11.1", features = ["storage"] }
```

Comment:
Feature sets are quite similar on all of the GCP implementations, but:

1. Developed by company rather than individual;
2. Have more than one maintainer/publisher;
3. Have plug-able http servers and auth flows;